include/mbedtls/ssl.h

```diff
@@ -1644,6 +1654,18 @@ struct mbedtls_ssl_context {
      */
     mbedtls_ssl_protocol_version MBEDTLS_PRIVATE(tls_version);
 
+#if defined(MBEDTLS_SSL_EARLY_DATA) && defined(MBEDTLS_SSL_CLI_C)
+    /**
+     *  Status of the negotiation of the use of early data.
+     *  See the documentation of mbedtls_ssl_get_early_data_status() for more
```

Comment:
Err, I can see the documentation for `mbedtls_ssl_get_early_data_status()` but I can't see the implementation - what am I missing?

Reply:
It is not implemented yet, like we have the documentation of mbedtls_ssl_write_early_data() but it is not implemented yet. It is a client specific API.